crates/swc_css_minifier/tests/fixture/compress-calc/media-query/output.min.css

```diff
@@ -0,0 +1 @@
+@media(min-width:calc(10px + 10px)){.class1{color:red}}
```

Comment:
Can you investigate why it was not compressed?

Reply:
Here what i figured out: for now, the "calc folding" work with enums like CalcValue::Dimension, CalcValue::Number, and so on. In the case of the "@media" rule, the AST contains other enums ; the Function value is a vector of PreservedToken.

What should we do:
* change the parser to have a Function with a vector of CalcValue enum?
* enhance the calc computation to work with the PreservedToken enum?

Reply:
It will be done here: https://github.com/swc-project/swc/pull/6140